src/app/content/reducer.ts

```diff
@@ -5,22 +5,26 @@ import pick from 'lodash/fp/pick';
 import { Reducer } from 'redux';
 import { getType } from 'typesafe-actions';
 import { ActionType } from 'typesafe-actions';
+import { receiveFeatureFlags } from '../actions';
 import { locationChange } from '../navigation/actions';
 import { matchForRoute } from '../navigation/utils';
 import { AnyAction } from '../types';
 import * as actions from './actions';
+import { studyGuidsFeatureFlag } from './constants';
 import highlightReducer, {initialState as initialHighlightState } from './highlights/reducer';
 import { content } from './routes';
 import searchReducer, {initialState as initialSearchState } from './search/reducer';
 import { State } from './types';
 
 export const initialState = {
+  enableStudyGuides: false,
   highlights: initialHighlightState,
   loading: {},
   params: null,
   references: [],
   search: initialSearchState,
   showCallToActionPopup: null,
+  studyGuides: null,
```

Comment:
this could be studyGuidesSummary or something

Reply:
Currently we are fetching only summary but in the end it will looks more like `highlights` so I think it's okay to name it `studyGuides` from the beginning